src/client/utils.js

```diff
@@ -1,7 +1,22 @@
-// log to Google Analytics
-export default function LogEvent(label) {
+const cagovPropertyId = "UA-3419582-2";
+const eddPropertyId = "UA-3419582-31";
```

Comment:
Would it make sense to store this variable in Azure, or is it ok that it's in a public repo?

Reply:
it's a public ID that viewable in the source for every website that uses google analytics, so it should be fine!